config/routes/spree.rb

```diff
@@ -59,7 +59,7 @@
 
     resources :products, except: :index do
       member do
-        get :clone
+        post :clone
```

Comment:
Clone wars! What a funny side effect. Are we sure we want to keep the pre-loading? Would that apply to pagination as well? The faster user experience comes at the cost of more server load and unnecessary requests.

Reply:
In this case, the pagination got implemented with a form submit, not links, so it doesn't apply there. But it does apply to the "discard changes" link, which reloads the product list.
We really don't want it firing off lots of expensive requests without good cause! 

In other cases, it's quite nice, eg when you click on New or Edit product, it can take you there instantly.

The handbook discusses some ways you can tweak this behaviour (https://turbo.hotwired.dev/handbook/drive#prefetching-links-on-hover), but probably the simplest and safest is to disable pre-fetch in the admin interface entirely for now. I will do that on the next revision.

Reply:
Really, I think pre-fetch should only be used on cacheable pages (and hopefully Turbo is smart enough not to re-fetch them). So it would be most useful on the frontend.

Reply:
> pre-fetch should only be used on cacheable pages

Totally agree. We should choose an opt-in approach.